recipe/bld.bat

```diff
@@ -1,14 +1,6 @@
-
-:: correct FC, apparently pointed to host prefix??
-set FC=%BUILD_PREFIX%\Library\bin\flang.exe
 set BLAS_ROOT=%PREFIX%
 set LAPACK_ROOT=%PREFIX%
-if exist %PREFIX%\Scripts\f2py.exe (
-  set F2PY=%PREFIX%\Scripts\f2py.exe
-) else (
-  set F2PY=%PREFIX%\Scripts\f2py.bat
-)
 
-"%PYTHON%" setup.py install
+"%PYTHON%" setup.py install -G "NMake Makefiles" -DBLA_VENDOR=Generic
```

Comment:
Why specify the vendor here?

Reply:
I just took the bld.bat currently in https://github.com/python-control/Slycot.

This flag enforces building against netlib no matter what, which is the recommended mode: https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

From https://github.com/python-control/Slycot/blob/master/conda-recipe/meta.yaml:

    # Always build against NETLIB ('Generic') LAPACK/Blas
    # https://conda-forge.org/docs/maintainer/knowledge_base.html#blas
    # deviating from above link: we have to specifiy netlib variant, because
    # the mkl variant selected by default for older pythons on windows
    # does not provide the generic headers